Engine/Alphas/StatisticsInsightManagerExtension.cs

```diff
@@ -125,8 +125,8 @@ public void OnInsightAnalysisCompleted(InsightAnalysisContext context)
                 var newMean = mean + (score - mean) / Statistics.TotalInsightsAnalysisCompleted;
                 Statistics.MeanPopulationScore.SetScore(scoreType, newMean, currentTime);
 
-                var newEma = score;
-                if (Statistics.TotalInsightsAnalysisCompleted > 1)
+                var newEma = mean;
```

Comment:
This should be `var newEma = newMean` -- and we need the statistics regenetated.